src/tooltip/tooltip.scss

```diff
@@ -14,7 +14,7 @@
   width: max-content;
   overflow-wrap: break-word;
   box-sizing: content-box;
-  padding: round($iui-baseline / 3) $iui-s;
+  padding: round($iui-baseline * 0.3333) $iui-s;
```

Comment:
And this

Reply:
1/3 = 0.33333

Reply:
So is it 4 or 5 symbols after coma? :D

Reply:
ditto here, I updated to 0.33333333333333333 to match values returned when dividing by 3

Reply:
oh...
@mayank99, could you quickly test from this branch generated css vs the current one if values produced are same?
Just to be sure that we need these "huge" numbers :D

Reply:
So I checked the generated css, and then realized we are using `round` here so the precision would not matter after like 5 points.

And in that other example, we have `calc` so I think we don't even need to switch to `*` (because for [calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()), `/` is a supported operator in native css).

Reply:
@mayank99 - so change it to 0.33?

Reply:
Yup

Reply:
OK, I think I have it cleaned up now.